lib/l10n/app_en.arb

```diff
@@ -143,10 +143,40 @@
-  "pageHelpWhatIsMotisAnswer": "With the MOTIS Ride App, you can do a lot of useful stuff!",
-  "pageHelpWhereIsAccessibilityQuestion": "Where are the accessibility settings?",
-  "pageHelpWhereIsAccessibilityAnswer": "This app supports a multitude of accessibility settings offered by the OS. Go to your OS's settings to enable Screen Reader or large fonts!",
+  "pageHelpEmailSubject": "Inquiry by User-ID [{id}]",
+  "@pageHelpEmailSubject": {
+    "placeholders": {
+      "id": {}
+    }
+  },
+  "pageHelpQuestion1": "What is MOTIS?",
+  "pageHelpAnswer1": "MOTIS encompasses many tools to support intermodal routing. Its goal is to help plan journeys that include public transport, bike sharing, ride sharing and more modes of transportation.",
+  "pageHelpQuestion2": "How much do rides cost? How can I pay?",
+  "pageHelpAnswer2": "We designed the payment process to fully rely on user agreements. Both the cost and payment method of a ride can be negotiated in person or in our in-app chat. You will see a price suggestion for most rides as a starting point.",
+  "pageHelpQuestion3": "We agreed on a price, but now my rider isn't paying it. What can I do?",
+  "pageHelpAnswer3": "We're sorry for the inconvenience. Unfortunately, the only thing we can offer you is the ability to report users in that case. Tap the \"Report user \" button on their profile and make sure to select the reason \"Did not pay the agreed amount\".",
+  "pageHelpQuestion4": "A driver/rider didn't show up. What can I do?",
+  "pageHelpAnswer4": "We're sorry for the inconvenience. Unfortunately, the only thing we can offer you is the ability to report users in that case. Tap the \"Report user \" button on their profile and make sure to select the reason \"Did not show up\".",
+  "pageHelpQuestion5": "My ride got cancelled. What can I do?",
+  "pageHelpAnswer5": "We're sorry for the inconvenience. Unfortunately, you will have to look for another ride. If the driver cancelled the drive just before the trip, you can report them using the \"Report user \" button on their profile. Make sure to select the reason \"Other\" and explain the situation.",
+  "pageHelpQuestion6": "My ride was rejected. What can I do?",
+  "pageHelpAnswer6": "We're sorry for the inconvenience. Unfortunately, you will have to look for another ride.",
+  "pageHelpQuestion7": "The driver isn't reacting to my request. What can I do?",
+  "pageHelpAnswer7": "We're sorry for the inconvenience. If you tap on the drive, you can take the request back using the button at the bottom. Then, you can look for a new ride.",
+  "pageHelpQuestion8": "My driver/riders are misbehaving. What can I do?",
+  "pageHelpAnswer8": "We're sorry for the inconvenience. We are not responsible for the actions of our users. If other users are using bad language, behaving aggressively or offensively or otherwise badly, please report them using the \"Report user \" button on their profile. Make sure to select the appropriate reason.",
```

Comment:
replace bad with inappropriate. and badly with inappropriately

Reply:
I think bad language is a fixed expression